transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/TransactionOutbox.java

```diff
@@ -370,24 +372,26 @@ private Instant after(Duration duration) {
   private void updateAttemptCount(TransactionOutboxEntry entry, Throwable cause) {
     try {
       entry.setAttempts(entry.getAttempts() + 1);
-      var blacklisted = entry.getAttempts() >= blacklistAfterAttempts;
-      entry.setBlacklisted(blacklisted);
+      var blocked = entry.getAttempts() >= blockedAfterAttempts;
+      entry.setBlocked(blocked);
       entry.setNextAttemptTime(after(attemptFrequency));
       validator.validate(entry);
       transactionManager.inTransactionThrows(transaction -> persistor.update(transaction, entry));
       listener.failure(entry, cause);
-      if (blacklisted) {
+      if (blocked) {
         log.error(
-            "Blacklisting failing process after {} attempts: {}",
+            "Marking entry {} as blocked after {} attempts: {}",
```

Comment:
Blocking failing process after {} attempts: {}

Reply:
You cool with `Blocking failing entry {} after {} attempts: {}` instead?